policyengine_us/variables/gov/states/ne/tax/income/credits/ctc/refundable/ne_refundable_ctc.py

```diff
@@ -0,0 +1,24 @@
+from policyengine_us.model_api import *
+
+
+class ne_refundable_ctc(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Nebraska refundable Child Tax Credit"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://nebraskalegislature.gov/laws/statutes.php?statute=77-7203"
+    )
+    defined_for = StateCode.NE
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.ne.tax.income.credits.ctc.refundable
+        # Total household income is defined as federal adjusted gross income
+        # as per Nebraska Department of Revenue
+        adjusted_gross_income = tax_unit("adjusted_gross_income", period)
```

Comment:
Could you make a new variable `ne_refundable_ctc_total_household_income` and point to AGI, with https://revenue.nebraska.gov/businesses/child-care-tax-credit-act as the reference? Avoids the need for the comment in two places (we'd also want the reference in both).